app/models/requisition.rb

```diff
@@ -1,4 +1,4 @@
-class ApplicantsPosition < ApplicationRecord
+class Requisition < ApplicationRecord
   belongs_to :applicant
   belongs_to :position
   enum status: [:applicant_interested, :currently_interviewing, :intend_to_hire]
```

Comment:
Also, what is the default? `nil`? If the requisition exists, it's presumably because an applicant is interested, so that would then make sense. From there, the status is elevated to something other than interest.